pkg/kubeclient/client_test.go

```diff
@@ -0,0 +1,106 @@
+package kubeclient_test
+
+import (
+	"context"
+
+	"github.com/kubernetes-sigs/kubebuilder/pkg/kubeclient"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ExampleClient_Create() {
+	kc, err := kubeclient.FromConfigOrCluster("/home/dir/config/.kubeconfig")
```

Comment:
does it have a default or should I always explicitly specify the location?

Reply:
Now config is passed instead, so this is handled by the caller.